src/effects/DynamicRangeProcessorHistoryPanel.cpp

```diff
@@ -26,7 +26,10 @@ namespace
 {
 constexpr auto defaultHeight = 100;
 constexpr auto timerId = 7000;
-constexpr auto timerPeriodMs = 1000 / 50;
+// Of course we aren't really targetting 200fps, but when specifying 50fps, we
+// rather get 30fps, with outliers at 20. Measurements (Windows) showed that,
+// when specifying 200, we get around 60fps on average, with outlier around 40.
+constexpr auto timerPeriodMs = 1000 / 200;
```

Comment:
Here some image for comparison: 
![Elapsed-time-comparison](https://github.com/audacity/audacity/assets/22740106/d43a9171-7bf7-436b-b4f8-8e70ab644b58)

Reply:
The x-axis is elapsed times between two ticks.

Reply:
`wxTimer` isn't precise mechanism indeed, alternatively you can try set 1ms update period and call `wxWindow::Refresh`(without `wxWindow::Update`) from timer update function making an early return from it until next `OnPaint`. I don't think it will yield better result, but then you don't have to explain numbers.
Btw, I checked Auburn Couture plugin and it's a bit jittery too.
👍

Reply:
Interesting idea. I just tried it, but indeed, no improvement.